acceptance/helper.rb

```diff
@@ -867,7 +868,12 @@ def install_puppet_from_package
       when :fedora
         on host, "yum install -y puppet-3.7.3"
       when :redhat
+        if options[:type] == 'aio' then
```

Comment:
So I guess this means we have a new 'type' that is different from 'foss' and 'pe' now? Thats what the variable had before wasn't it?